src/parser/hunter/shared/modules/talents/AMurderOfCrows.js

```diff
@@ -66,7 +68,8 @@ class AMurderOfCrows extends Analyzer {
       // If more than 1 second has passed and less than the duration has elapsed, we can assume that crows has been reset, and thus we reset the CD.
       this.spellUsable.endCooldown(SPELLS.A_MURDER_OF_CROWS_TALENT.id, event.timestamp);
       this.maxCasts += 1;
-      debug && console.log("Crows was reset at: ", event.timestamp);
+      this.resets += 1;
+      debug && console.log('Crows was reset at: ', event.timestamp);
```

Comment:
I think zero wants all instances of console.log replace with this.log (or this.debug, etc); which adds the timestamp and other meta information automatically

```suggestion
      debug && this.log('Crows was reset');
```